feature/app.js

```diff
@@ -0,0 +1,71 @@
+const createError = require('http-errors');
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+const mongoose = require('mongoose');
+
+const indexRouter = require('./routes/index');
+// const usersRouter = require('./routes/users');
+// const labelRouter = require('./routes/label');
```

Comment:
PR should only consist of code related to targeted issue.

Reply:
File updated.